doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1450 @@
+.. _ncs_release_notes_3.0.0:
+
+|NCS| v3.0.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This major release of |NCS| introduces significant changes that can break backward compatibility for existing projects.
+Refer to the :ref:`migration_3.0` for further information.
+
+The SDK can be installed using any one of the following options:
+
+* Using |VSC| and the :ref:`requirements_nrfvsc` (recommended)
+* Using command line and :ref:`requirements_nrf_util`
+
+Starting with this release, the SDK installation using the Toolchain Manager app is not supported.
+
+Added the following features as supported:
+
+* Bluetooth®:
+
+  * Channel sounding Controller and Host will be Bluetooth qualified for the nRF54L Series.
+  * The :ref:`channel_sounding_ras_initiator` sample now includes an IFFT algorithm for Phase-Based Ranging, providing a viable distance measurement result for basic ranging applications.
+    Results from the illustrative MCPD algorithm from the |NCS| 2.9.0 release, which is shown alongside the IFFT result, are still not recommended for ranging.
+  * Bluetooth controller and host qualified for the nRF54H20.
+
+* Matter:
+
+  * Matter 1.4.1: Matter commissioning using NFC tag can be officially certified now.
+  * :ref:`matter_manufacturer_specific_sample` sample: Dedicated sample, containing documentation and a preview :ref:`PC tool <ug_nrfconnect_manufacturer_cluster_editor_tool>`, facilitating creation and modification of manufacturer specific clusters.
+
+* PMIC:
+
+  * nPM2100 and nPM2100 EK:
+
+    * Added support for nPM2100, which is a PMIC designed for primary (non-rechargeable) batteries in an extremely compact form factor.
+      It has an ultra-efficient boost regulator, a dual purpose LDO/load switch, two GPIOs, an ADC, and other features.
+    * :ref:`npm2100_fuel_gauge` sample, demonstrating how to calculate the state of charge of a supported primary cell battery using the nPM2100 and the :ref:`nrfxlib:nrf_fuel_gauge`.
+    * :ref:`npm2100_one_button` sample, demonstrating how to support wake-up, shutdown, and user interactions through a single button connected to the nPM2100.
+
+* nRF54L Series:
+
+  * Bootloader and Device Firmware Update (DFU):
+
+    * nRF Secure Immutable Bootloader. (to be updated)
+    * MCUboot encrypted DFU using ECIES.(to be updated)
```

Comment:
Having these features as supported seems to contradict the SML tables:

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/software_maturity.html#nrf_secure_immutable_bootloader

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/software_maturity.html#mcuboot_bootloader

CC: @nvlsianpu

Reply:
These are now moved to experimental (as agreed with PMT)